docs/projects/even-odd/_posts/2018-12-31-python.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Even-Odd in Python
+layout: default
+last-modified: 2018-12-31
+tags: ['python', 'even-odd', 'syntax']
+authors: "Chase Stevens"
+---
+
+# Even_Odd
+
+Oftentimes when dealing with things and numbers, we want to split up what we're working with into two groups. This property, known as parity, is useful to know. With Python, we can create a function to check the parity of any given integer.
+
+    def even_odd(x):
+        return "Even" if x % 2 == 0 else "Odd"
+
+Here we have a function named 'even_odd' that takes the parameter "x". The function checks to see if x % 2 is equal to 0. If this is true, we return the value "Even", and if it is false, then we return "Odd."
+
+### Modulo
+One symbol which sticks outs is "%". In Python, and in many other programming languages, the symbol % represents a "modulo" operation, which returns the remainder of the first number divided by the second number.
+
+Let's say we wanted to find out the remainder of 5 divided by 2. We'd calculate that 2 goes into 5 twice with a remainder of one. We can express this with the modulo operator as 5 % 2 = 1. Similarly, we could check the modulo of 9 and 3. Since 3 goes into 9 evenly, we would get an answer of 0, expressed as 9 % 3 = 0.
+
+Since an integer is even if it is divisible by two, we can check to see if a number would have any remainder after being divided by two -- or take a number mod two and if that's equal to zero, we know that said number is even.
+
+### Syntactic Sugar
+
+Some may look at the code and notice that our function fits nearly on a single line. However, we have fit a conditional statement that returns multiple values -- how did we do that?
+
+We could also write our function this way.
+
+    def even_odd(x):
+        if x % 2 == 0:
+    	return "Even"
+        else:
+    	return "Odd"
+
+Here we have our conditional statement as well as our two return values clearly defined. While it certainly works, it's not as elegant as our original function.
+
+Syntactic sugar allows us to take four lines of code and reduce it to one while improving our code's readability -- not a bad improvement!
+
+## How to run the solution
+
+Download the file, and from the command line, enter "python even-odd.py [your number here]".
+
+Alternatively, you can see the code in action at this online [Python interpreter](https://trinket.io/python3/c2ed217a30?runMode=console).
```

Comment:
While I'm sure this is a good online interpreter, in order to maintain consistency, it might be wise to reference the online interpreter that's already in our python README. Or if this is a better one, it could be added to the README. @jrg94, thoughts?

Reply:
I don't have an opinion either way. On one hand, more resources are good and the one in the README is likely arbitrary. On the other hand, consistency is good. I'm happy with the fix!